proto/kvrpcpb.proto

```diff
@@ -686,9 +686,9 @@ message Context {
     // Any additional serialized information about the request.
     bytes resource_group_tag = 18;
 
-    // Flag used to tell tikv that operations can be allowed even when tikv disk full happens.
-    bool allowed_on_disk_full = 19;
-
+    // Used to tell TiKV what operations are allowed when the disk status is at different levels.
+    AllowedLevel allowed_level = 19;
```

Comment:
This is too ambiguous. We can tell what is it used for from the name.

Reply:
Maybe consider `DiskFullOpt`

Reply:
changed